### PR TITLE
lesson 1 translated

### PR DESCRIPTION
**Pull request by SeanICho**
- [ V ] I did these translations myself and own copyright for them
- [ V ] I didn't use a machine to do these translations
- [ V ] I assign all copyright to Loom Network for these translations

### CODE REVIEW

File: ko/share-template-msgs.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "1": {
```
Comment (mattkanwisher) [inline ko/share-template-msgs.json]: This needs to be in Korean

File: ko/index.json [hunk 2]
```diff
@@ -0,0 +1,197 @@
+{
+    "base": {
```
Comment (mattkanwisher) [inline ko/index.json]: This needs to be in Korean

File: ko/share-template-msgs.json (hunk 1 above)
Comment (lukezhangstudio) [reply ko/share-template-msgs.json]: what matt means is the words that follow this. not the keys.

`"key": "translate this stuff"`

File: ko/index.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    arraysstructs2,
+    functions2,
+    functions3,
+    keccak256,
+    puttingittogether,
+    events,
+    web3js,
+    lessoncomplete
+  ],
+  2: [
```
Comment (lukezhangstudio) [inline ko/index.ts]: you should comment out lesson 2 and 3 until they are translated.

**Pull request closed without merging**